nnvm/include/nnvm/op.h

```diff
@@ -340,6 +340,13 @@ class OpMap {
    */
   inline int count(const Op* op) const;
 
+  /*!
+   * \brief Check if the map has op as key.
+   * \param op The key to the map
+   * \return true if op is contained in map, false otherwise.
+   */
+  inline bool contains(const Op* op) const;
```

Comment:
contains is just same as count so it is not necessary to add this function.

Reply:
C++20 adds this function, I think is more readable, would be good to have it as well, don't you think is nice?

https://en.cppreference.com/w/cpp/container/unordered_map

Reply:
@tqchen ping